build.sbt

```diff
@@ -156,29 +155,24 @@ lazy val fatJarSettings = Seq(
   }
 )
 
+git.uncommittedSignifier := Some("dirty")
```

Comment:
if these settings are used only for docker build, maybe we can scope and move them to `dockerSettings`?

Reply:
Done, please check it now.